### ISSUE TITLE
Model Training 

### ISSUE THREAD

**Issue by shreeyanshi123** [opencodeiiita/DeepSeaROV#29]
1.Model-Training:
●	Choose a suitable architecture (e.g., CNN, transfer learning with pre-trained models).
●	Use appropriate loss functions and optimizers for multi-class classification.
2. Evaluation
●	Generate predictions on the test dataset.
●	Plot the ROC Curve for each class.
  
Add your notebook and add your saved model  in solution_Task3 . 

This is a competitive task and only the best model will get 50points others will get 20points.

**Comment by UrvashiPandey14** [comment]
Hi! I would like to work on this issue. Can you please assign it to me? @shreeyanshi123
